prediction-applier-extension/src/main/resources/alfresco/module/alfresco-hxinsight-connector-prediction-applier-extension/model/hxi-model.xml

```diff
@@ -15,10 +15,96 @@
         <namespace uri="http://www.alfresco.org/model/hxi/1.0" prefix="hxi"/>
     </namespaces>
 
+    <constraints>
+        <constraint name="hxi:updateTypeList" type="LIST">
+            <parameter name="allowedValues">
+                <list>
+                    <value>AUTOFILL</value>
+                    <value>AUTOCORRECT</value>
+                </list>
+            </parameter>
+            <parameter name="caseSensitive">
+                <value>false</value>
+            </parameter>
+        </constraint>
+    </constraints>
+
+    <types>
+        <type name="hxi:prediction">
+            <title>Prediction</title>
+            <parent>cm:cmobject</parent>
+
+            <properties>
+                <property name="hxi:predictionDateTime">
+                    <title>Prediction Date Time</title>
+                    <description>Time that the prediction was generated</description>
+                    <type>d:datetime</type>
+                    <mandatory>true</mandatory>
+                </property>
+                <property name="hxi:confidenceLevel">
+                    <title>Confidence Level</title>
+                    <description>Prediction confidence from Hx Insight</description>
+                    <type>d:float</type>
+                    <mandatory>true</mandatory>
+                </property>
+                <property name="hxi:modelId">
+                    <title>Prediction Model Id</title>
+                    <description>Identified for the model used by Hx Insight</description>
+                    <type>d:text</type>
+                    <mandatory>true</mandatory>
+                </property>
+                <property name="hxi:predictionValue">
+                    <title>Prediction Value</title>
+                    <description>Value predicted by Hx Insight</description>
+                    <type>d:any</type>
```

Comment:
Why to use ANY instead of NODE_REF?
Is the value stored for ANY properties searchable in Solr/Elasticsearch?

Reply:
The value here could be a boolean, number, string, etc. This is the predicted value from Hx Insight.